unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Interfaces/IParcelScene.cs

```diff
@@ -19,6 +19,7 @@ public interface IParcelScene
         Dictionary<long, IDCLEntity> entities { get; }
         IECSComponentsManagerLegacy componentsManagerLegacy { get; }
         LoadParcelScenesMessage.UnityParcelScene sceneData { get; }
+        HashSet<Vector2Int> parcels { get; }
```

Comment:
doesn't scenedata already contains a list of parcels?

Reply:
Yes but sceneData has a `Vector2int[]`, and the hashset is used to quickly find if a coordinate is inside the parcels in the original [`IsInsideSceneBoundaries(Vector3 worldPosition, float height = 0f)`](https://github.com/decentraland/unity-renderer/blob/f1a32acd27431f5bee71bc1fb748993954747f2e/unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/ParcelScene.cs#L282)

Reply:
I'm not a big fan of this interface keeping growing plus it's the same data already available but with a different collection type.
so I would prefer that if a particular system, for performance porpoises, needs a hashset, then we should make that conversion on the system itself

Reply:
Yes but since the SDK6 approach still uses that, I can't remove it, otherwise if I add the new one leaving the existent one we would have 3 collections with the same parcels now (1 array in the sceneData, 1 hashset in the ParcelScene, 1 hashset in the SBC System)...

Reply:
I'll try to move that to an external scenes dictionary

Reply:
Check now